specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/extensions.json

```diff
@@ -0,0 +1,346 @@
+                "x-ms-examples": {
+                    "Enable cluster monitoring": {
+                        "$ref": "./examples/DisableLinuxClusterMonitoring.json"
+                    }
+                },
+                "operationId": "Extension_DisableMonitoring",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Ok response definition."
+                    },
+                    "202": {
+                        "description": "OK response definition."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+	
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/extensions/{extensionName}": {
+            "put": {
+                "tags": [
+                    "Extensions"
+                ],
+                "description": "Creates an HDInsight cluster extension.",
+                "operationId": "Extension_Create",
+                "x-ms-examples": {
+                    "Create a monitoring extension on Hadoop Linux cluster": {
+                        "$ref": "./examples/CreateExtension.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ExtensionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/Extension"
+                        },
+                        "description": "The cluster extensions create request."
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "OK. The request has succeeded."

```

Comment:
202 -> Does this PUT an LRO? Does it have 200 return code as well?

Reply:
This is a bug. enabling/disabling monitoring should match enabling/disabling extensions.